src/explore-education-statistics-common/src/modules/charts/components/MapBlockInternal.tsx

```diff
@@ -60,9 +59,8 @@ export interface MapBlockProps extends ChartProps {
     major: AxisConfiguration;
   };
   boundaryLevel?: number;
-  customDataGroups?: CustomDataGroup[];
   dataGroups?: number;
-  dataClassification?: DataClassification;
+  dataClassification?: DataGroupingType;
```

Comment:
Would suggest creating a ticket for removing these and migrating over to the new map `dataSetConfigs`.

Should have comments referencing this ticket as well.

Reply:
I created the ticket to reference on another PR - the description still needs filing out.